views/confirmation-still-required.html

```diff
@@ -48,6 +48,15 @@ <h2 class="govuk-heading-m">What happens next</h2>
             but it must file its {{ overdueFiling }} to stay on the register.
         </p>
 
+        <!-- This block is displayed if ONLY the confirmation statement is late: -->
```

Comment:
Would these comments be visible on the source for the actual page? May be better to put the comments in nunjucks so they aren't sent to the user
https://mozilla.github.io/nunjucks/templating.html#comments

Reply:
Yes, visible in the source so will do as suggested.